crates/rpc/rpc/src/debug.rs

```diff
@@ -332,7 +341,7 @@ where
                                 Ok(frame.into())
                             })
                             .await?;
-                        return Ok(frame)
+                        return Ok(frame);
```

Comment:
could you please undo these, makes it harder to review, idk why they keep reappearing...

Reply:
when you run `cargo fmt` and not `cargo +nightly fmt`...doom

Reply:
Oopsie, will update them

Reply:
About the approach does modifying the `trace_transaction` function to accept an optional inspector and return it along with the result and then modify `trace_block` function to use it sound good?